src/configuration/argv_parser.ts

```diff
@@ -65,7 +64,7 @@ const ArgvParser = {
   },
 
   parse(argv: string[]): IParsedArgv {
-    const program = new Command(path.basename(argv[1]))
+    const program = new Command('cucumber-js')
```

Comment:
Is this expected rather than `cucumber.js`?
If so, what's that mean? Will we still call cucumber through `cucumber-js` command?

Reply:
Yeah, the command is still `cucumber-js`, the way it's called hasn't changed (because it's done via symlinks on install), it's just the name of the underlying file.

Reply:
Ok, thanks for the info 👌